juriscraper/scotus/scotus_email.py

```diff
@@ -173,6 +231,12 @@ def _parse_docket_number(self) -> str:
         docket_number = Path(file).stem
         return clean_string(docket_number)
 
+    def _parse_confirmation_link(self) -> str:
+        """Extract the `href` attribute from the first `<a>` tag in the email
+        body, which should be the confirmation link.
+        """
+        return self.tree.find(".//a").get("href")
```

Comment:
This has the same functionality as the `_parse_case_link` helper. I suggest using a more generic name like `_parse_first_link` and then reusing it in both cases.